hummingbot/strategy/liquidity_mining/liquidity_mining.py

```diff
@@ -100,59 +108,105 @@ def tick(self, timestamp: float):
-
-    def market_status_df(self) -> pd.DataFrame:
+    def budget_status_df(self) -> pd.DataFrame:
         data = []
-        columns = ["Exchange", "Market", "Mid Price", "Volatility", "Base Bal", "Quote Bal", " Base %"]
-        balances = self.adjusted_available_balances()
+        columns = ["Market", f"Budget({self._token})", "Base bal", "Quote bal", "Base/Quote"]
         for market, market_info in self._market_infos.items():
-            base, quote = market.split("-")
             mid_price = market_info.get_mid_price()
             base_bal = self._sell_budgets[market]
             quote_bal = self._buy_budgets[market]
-            total_bal = (base_bal * mid_price) + balances[quote]
-            base_pct = (base_bal * mid_price) / total_bal if total_bal > 0 else s_decimal_zero
+            total_bal_in_quote = (base_bal * mid_price) + quote_bal
+            total_bal_in_token = total_bal_in_quote
+            if not self.is_token_a_quote_token():
+                total_bal_in_token = base_bal + (quote_bal / mid_price)
+            base_pct = (base_bal * mid_price) / total_bal_in_quote if total_bal_in_quote > 0 else s_decimal_zero
+            quote_pct = quote_bal / total_bal_in_quote if total_bal_in_quote > 0 else s_decimal_zero
             data.append([
-                self._exchange.display_name,
                 market,
-                float(mid_price),
-                "" if self._volatility[market].is_nan() else f"{self._volatility[market]:.2%}",
+                float(total_bal_in_token),
                 float(base_bal),
                 float(quote_bal),
-                f"{base_pct:.0%}"
+                f"{base_pct:.0%} / {quote_pct:.0%}"
+            ])
+        df = pd.DataFrame(data=data, columns=columns).replace(np.nan, '', regex=True)
+        df.sort_values(by=["Market"], inplace=True)
+        return df
+
+    def market_status_df(self) -> pd.DataFrame:
+        data = []
+        columns = ["Market", "Mid price", "Best bid", "Best ask", "Volatility"]
+        for market, market_info in self._market_infos.items():
+            mid_price = market_info.get_mid_price()
+            best_bid = self._exchange.get_price(market, False)
+            best_ask = self._exchange.get_price(market, True)
+            best_bid_pct = abs(best_bid - mid_price) / mid_price
+            best_ask_pct = (best_ask - mid_price) / mid_price
+            data.append([
+                market,
+                float(mid_price),
+                f"{best_bid_pct:.2%}",
+                f"{best_ask_pct:.2%}",
+                "" if self._volatility[market].is_nan() else f"{self._volatility[market]:.2%}",
             ])
-        return pd.DataFrame(data=data, columns=columns).replace(np.nan, '', regex=True)
+        df = pd.DataFrame(data=data, columns=columns).replace(np.nan, '', regex=True)
+        df.sort_values(by=["Market"], inplace=True)
+        return df
+
+    async def miner_status_df(self) -> pd.DataFrame:
```

Comment:
Small matter. I think a better name for `miner_status_df` is `campaign_status_df`.

Reply:
This is to keep it inline with `status` as it is under `Miner` section.